src/acl/acl-control.ts

```diff
@@ -9,9 +9,9 @@ import utils from '../utils.js'
 import { ACLbyCombination, adoptACLDefault, getACLorDefault, makeACLGraphbyCombo, readACL, sameACL } from './acl'
 import widgets from '../widgets'
 import icons from '../iconBase.js'
-import authn from '../signin.js'
 import { graph, IndexedFormula, NamedNode, sym, UpdateManager } from 'rdflib'
 import { DataBrowserContext } from 'pane-registry'
+import { logInLoadProfile } from '../authn/authn'
```

Comment:
And authn/authn is messy -- what else will there be in authn/ apart from authn ?

Reply:
I was thinking `authentication` is a good name - is there a need to make it so short as `authn`?

Reply:
But remember, we need to keep the public API for this (this PR is only for migrating to TS), so we don't want to change the public name for this (yet at least)

Reply:
I think this concern is considered in https://github.com/solid/solid-ui/issues/120, and I will resolve this conversation so that we can merge this PR.